couchdb/src/main/java/org/hibernate/ogm/datastore/couchdb/CouchDBDialect.java

```diff
@@ -71,9 +71,9 @@ public CouchDBDialect(CouchDBDatastoreProvider provider) {
 
 	@Override
 	public Tuple getTuple(EntityKey key, TupleContext tupleContext) {
-		EntityDocument entity = getDataStore().getEntity( Identifier.createEntityId( key ) );
-		if ( entity != null ) {
-			return new Tuple( new CouchDBTupleSnapshot( entity.getProperties() ) );
+		EntityDocument entity = getDataStore().getEntity(Identifier.createEntityId(key));
```

Comment:
@wokier we are a bit weird in that way but we put spaces between parenthesis and what's inside parenthesis (except for method definition, catch and for I think). run mvn checkstyle:checkstyle, you will see these rules being raised.